boutiques_descriptors/explore-asl.json

```diff
@@ -3,17 +3,19 @@
     "description": "A pipeline and toolbox for image processing and statistics of arterial spin labeling perfusion MR images.",
```

Comment:
Should we rename it according to convention we discussed, if yes it will became `ExploreASL-BIDS_Subject`.